### PR TITLE
Add ability to delete observing runs

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds ability to delete observing runs.

### CODE REVIEW

File: skyportal/handlers/api/observingrun.py [hunk 1]
```diff
@@ -227,7 +227,7 @@ def put(self, run_id):
         self.push_all(action="skyportal/FETCH_OBSERVING_RUNS")
         return self.success()
 
-    @permissions(["Upload data"])
+    @auth_or_token
```
Comment (guynir42) [inline skyportal/handlers/api/observingrun.py:230]: Why is this one not with a "permissions"?

File: skyportal/utils/offset.py [hunk 2]
```diff
@@ -478,6 +478,14 @@ def _calculate_best_position_for_offset_stars(
         log("Uncaught error in ra, dec determination")
         return fallback
 
+    if np.isnan(med_ra) or np.isnan(med_dec):
+        log(
+            "Warning: the median of the positions"
+            " from the photometry data associated with this source "
+            " retured nan, using fallback"
+        )
+        return fallback
```
Comment (guynir42) [inline skyportal/utils/offset.py:487]: This seems unrelated, but I guess it's good to have!

File: static/js/components/ObservingRunPage.jsx [hunk 3]
```diff
@@ -115,14 +156,23 @@ const ObservingRunList = ({ observingRuns }) => {
 
 const ObservingRunPage = () => {
   const { observingRunList } = useSelector((state) => state.observingRuns);
```
Comment (guynir42) [inline static/js/components/ObservingRunPage.jsx:158]: Are you sure you want to call the variable holding the observing runs `observingRunList` the same as the name of one of the components? maybe just use `observingRuns`?

**Review by guynir42 — APPROVED** [review]
Minor suggestions.

File: skyportal/handlers/api/observingrun.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/handlers/api/observingrun.py:230]: The permissions are done in the handler (we also do this for others)

File: skyportal/utils/offset.py (hunk 2 above)
Comment (mcoughlin) [reply skyportal/utils/offset.py:487]: yeah just something.i noticed.

File: static/js/components/ObservingRunPage.jsx (hunk 3 above)
Comment (mcoughlin) [reply static/js/components/ObservingRunPage.jsx:158]: It is already used in other places so probably too late to switch now.

File: skyportal/handlers/api/observingrun.py (hunk 1 above)
Comment (guynir42) [reply skyportal/handlers/api/observingrun.py:230]: OK just checking.

File: static/js/components/ObservingRunPage.jsx (hunk 3 above)
Comment (guynir42) [reply static/js/components/ObservingRunPage.jsx:158]: Seems very confusing to me, but fine.

**Pull request merged**